src/Player.js

```diff
@@ -29,7 +29,7 @@ class Player {
             debug: true,
             latitude: 0,
             longitude: 0,
-            altitude: 0,
+            accuracy: 23,

```

Comment:
An high accuracy makes API returns 0 pokemons nearby, setting it to 0 seems to work perfectly